requirements/prod.in

```diff
@@ -9,6 +9,6 @@ python-dateutil
 gvgen
 pydantic
 typing_extensions
-python-baseconv
-backports-datetime-fromisoformat        # for Python 3.6
+python-baseconv 
+backports-datetime-fromisoformat      ; python_version < '3.7'
```

Comment:
I think this will still make it into install_requires in setup.py because our logic there is ignoring anything after a space: https://github.com/SFDO-Tooling/Snowfakery/blob/main/setup.py#L12